projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -141,6 +141,21 @@ function process_anchor_params() {
 							),
 						);
 					}
+				} else {
+					$retry_url         = add_query_arg(
```

Comment:
were those spaces added on purpose?

Reply:
Looks like it's being added on commit by PHPCBF.

Reply:
👍